kimchi/src/circuits/expr.rs

```diff
@@ -508,6 +508,7 @@ pub enum FeatureFlag {
     ForeignFieldAdd,
     ForeignFieldMul,
     Xor,
+    Rot,
```

Comment:
For simplicity and consistency, I feel like these should match the `GateType` names

Reply:
well, rangecheck does not come with a 0 or a 1, nor chacha has the 0 1 final, I think FeatureFlag just needs to be readable for us, no need to be so specific on which exactly. If we ever have a rot128, we would still want the feature flag to be general for rotation, and not tied to 64 or whatever